packages/core-utils/src/itinerary.js

```diff
@@ -159,51 +181,6 @@ export function getMapColor(mode) {
   return "#aaa";
 }
 
-// TODO: temporary code; handle via migrated OTP i18n language table
-export function getStepDirection(step) {
-  switch (step.relativeDirection) {
-    case "DEPART":
-      return `Head ${step.absoluteDirection.toLowerCase()}`;
-    case "LEFT":
-      return "Left";
-    case "HARD_LEFT":
-      return "Hard left";
-    case "SLIGHTLY_LEFT":
-      return "Slight left";
-    case "CONTINUE":
-      return "Continue";
-    case "SLIGHTLY_RIGHT":
-      return "Slight right";
-    case "RIGHT":
-      return "Right";
-    case "HARD_RIGHT":
-      return "Hard right";
-    case "CIRCLE_CLOCKWISE":
-      return "Follow circle clockwise";
-    case "CIRCLE_COUNTERCLOCKWISE":
-      return "Follow circle counterclockwise";
-    case "ELEVATOR":
-      return "Take elevator";
-    case "UTURN_LEFT":
-      return "Left U-turn";
-    case "UTURN_RIGHT":
-      return "Right U-turn";
-    default:
-      return step.relativeDirection;
-  }
-}
-
-export function getStepInstructions(step) {
-  const conjunction = step.relativeDirection === "ELEVATOR" ? "to" : "on";
-  return `${getStepDirection(step)} ${conjunction} ${step.streetName}`;
-}
-
-export function getStepStreetName(step) {
-  if (step.streetName === "road") return "Unnamed Road";
-  if (step.streetName === "path") return "Unnamed Path";
-  return step.streetName;
-}
-
 export function toSentenceCase(str) {
```

Comment:
This one might need to be deprecated.

Reply:
There are a bunch of otp-rr keys that rely on this method to be generated. We could go through and replace all those keys, but we'd still need some form of string recapitalization/normalization. What do you think? The only other use from within otp-ui is in a deprecated method